src/cli/config/init.ts

```diff
@@ -90,6 +90,7 @@ export const run: CliCommand = async (args: Arguments /* , logger: Logger */) =>
     if (!jestPreset) {
       content.push(`${preset.jsImport('tsjPreset')};`, '')
     }
+    content.push(`/** @type {import('@jest/types').Config.InitialOptions} */`)
```

Comment:
you should use `/** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */` which also types `ts-jest` options.

Reply:
Awesome! Will do!